xcode/Subconscious/Shared/Components/Common/SubtextTextViewRepresentable.swift

```diff
@@ -287,44 +250,6 @@ struct SubtextTextViewRepresentable: UIViewRepresentable {
                 )
             )
         }
-        
-        // MARK: - NSTextLayoutManagerDelegate
-                                
-        func textLayoutManager(
```

Comment:
Given we're remove this function, how are we handling what it did before?

Reply:
None of this is actually being used, it's all obsolete residue from implementing the TextKit 2 delegate methods. The entire method is a no-op in the current build.